### PR TITLE
Add staging test-application in JPA Query bucket for future test dev.

### PR DESCRIPTION
**Pull request by jgrassel**
Adding a "jpql" test application group under the JPA query bucket, with the intent of it being a staging area for future test development.  The idea is to have a ready workspace that allows for immediate test development without wasting the time for the broiler plate.

### CODE REVIEW

File: dev/com.ibm.ws.jpa_spec10_query_fat/fat/src/com/ibm/ws/jpa/spec10/query/jpql/TestQueryJPQL_Web.java [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+            ddlSet.add(ddlName.replace("${dbvendor}", getDbVendor().name()));
+        }
+        executeDDL(server, ddlSet, false);
+
+        setupTestApplication();
+    }
+
+    private static void setupTestApplication() throws Exception {
+        WebArchive webApp = ShrinkWrap.create(WebArchive.class, appName + ".war");
+        webApp.addPackages(true, "com.ibm.ws.jpa.query.sqlcapture");
```
Comment (aguibert) [inline dev/com.ibm.ws.jpa_spec10_query_fat/fat/src/com/ibm/ws/jpa/spec10/query/jpql/TestQueryJPQL_Web.java]: the `true` boolean here means automatically adding sub-packages. Would it be simpler to just do:
`webApp.addPackages(true, "com.ibm.ws.jpa.query");`

File: dev/com.ibm.ws.jpa_spec10_query_fat/test-applications/query/jpql/web/jpql.war/WEB-INF/ibm-web-bnd.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+-->
+<web-bnd xmlns="http://websphere.ibm.com/xml/ns/javaee"
```
Comment (aguibert) [inline dev/com.ibm.ws.jpa_spec10_query_fat/test-applications/query/jpql/web/jpql.war/WEB-INF/ibm-web-bnd.xml]: this file probably isn't necessary and can be removed

File: dev/com.ibm.ws.jpa_spec10_query_fat/fat/src/com/ibm/ws/jpa/spec10/query/jpql/TestQueryJPQL_Web.java (hunk 1 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_spec10_query_fat/fat/src/com/ibm/ws/jpa/spec10/query/jpql/TestQueryJPQL_Web.java]: Ah, good point there.

**Comment by jgrassel** [comment]
#build

**Pull request merged**